tensorboard/uploader/uploader_test.py

```diff
@@ -1039,6 +1039,11 @@ def test_break_at_scalar_point_boundary(self):
 
         self.assertGreater(len(requests), 1)
         self.assertLess(len(requests), point_count)
+        # This is the observed number of requests when running the test. There
+        # is no reasonable way to derive this value from just reading the code.
+        # The number of requests does not have to be 33 to be correct but if it
+        # changes it probably warrants some investigation or thought.
+        self.assertEqual(33, len(requests))
```

Comment:
I found this check reassuring while refactoring the code but I recognize it's a weird magic number.  Hopefully it's continued existence is justified by the comment.

Reply:
ok with me